README.md

```diff
@@ -61,6 +61,8 @@ npm install systemjs
 * [API](docs/api.md)
 * [System.register](docs/system-register.md)
 * [Loader Hooks](docs/hooks.md)
+* [JSON modules](docs/json-modules.md)
+* [CSS modules](docs/css-modules.md)
```

Comment:
Instead of making these separate pages, perhaps we can have a single `docs/module-types.md` which has a linkable section for these. We could still have them as separate items in this list then.